docs/podman.1.md

```diff
@@ -209,13 +209,14 @@ Or just add the content manually.
 	$ echo USERNAME:10000:65536 >> /etc/subuid
 	$ echo USERNAME:10000:65536 >> /etc/subgid
 
+See man pages `subuid(5)` and `subgid(5)` for mor information.
```

Comment:
'mor information' to 'more information.

Suggest:
See the `subuid(5)` and `subgid(5)` man pages for more information.

Reply:
Fixed